configure-git/README.md

```diff
@@ -1,28 +1,39 @@
 # Git Kata: Configuring git
 
+This kata has no `setup.sh` script. Just read and follow along.
 
 ## 1. Git Initial Configuration
+
+Git wants to know who it should write as the committer of changes etc.
+In order to do that, configure user name and user email towards git with the following commands:
+
 1. `git config --global user.name "John Doe"`
 2. `git config --global user.email "johndoe@example.com`
 
-For the vim scared:
+### Editor setup
+
+Sometime git needs you to edit a file it creates e.g. the message of a commit you create.
+As a standard, it is configured with VIM, but that has a steep learning curve, so you might be better of with another tool of yor liking:
+
+If you want to use the cli based editor nano:
 - `git config --global core.editor nano`
 
 For the windows peeps:
 - `git config --global core.editor notepad`
 
-or alternatively:
+or alternatively other tools you already are familiar with:
 
 - `git config --global core.editor "'C:/Program Files/Notepad++/notepad++.exe' -multiInst -notabbar -nosession -noPlugin"`
 - `git config --global core.editor "atom --wait"`
 - `git config --global core.editor "code --wait"`
```

Comment:
VSCode has been added as an option during installation :)

Reply:
Yes, but what if you have not made that choice while installing? Also not part of the PR :)